### PR TITLE
Support refs without labels (a.k.a. releases without descriptions)

### PR DESCRIPTION
**Pull request by brettchalupa**
Bug scenario from @faresd:

> If you create a Release without a description, which is possible by
> Publishing a document and choosing "Publish at a specific date and
> time", the ref returned by Prismic's API will have a null label and the
> Prismic client will fail to parse the entire response.

Because of how the hash construction works, it needs a value for the
key. This makes it fall back to the id of the ref which should always be
present in the scenario where the label is null/nil.

fixes https://github.com/prismicio-community/ruby-kit/issues/116

More details:

- https://community.prismic.io/t/release-without-description-causes-error-in-prismic-ruby-client/7514/4

### CODE REVIEW

File: lib/prismic/api.rb [hunk 1]
```diff
@@ -232,7 +232,7 @@ def self.parse_api_response(data, access_token, http_client, cache)
         }]
         api.refs = Hash[data_refs.map { |ref|
           scheduled_at = ref['scheduledAt']
-          [ref['label'].downcase, Ref.new(ref['id'], ref['ref'], ref['label'], ref['isMasterRef'], scheduled_at && Time.at(scheduled_at / 1000.0))]
+          [ref['label']&.downcase || ref['id'].downcase, Ref.new(ref['id'], ref['ref'], ref['label'], ref['isMasterRef'], scheduled_at && Time.at(scheduled_at / 1000.0))]
```
Comment (alistairholt) [inline lib/prismic/api.rb:235]: Thanks for this. We can clean this up a little more:

```suggestion
          [ref.fetch('label', ref['id']).downcase, Ref.new(ref['id'], ref['ref'], ref['label'], ref['isMasterRef'], scheduled_at && Time.at(scheduled_at / 1000.0))]
```

**Review by alistairholt — CHANGES_REQUESTED** [review]
Thank you! Just a small tweak.

**Review by alistairholt — APPROVED** [review]

**Comment by brettchalupa** [comment]
You're welcome, thank you! :raised_hands: Do you know if this will be included in a new gem release?

**Comment by alistairholt** [comment]
@brettchalupa yes, it will be. I'll aim to get it out as soon as we can.

**Comment by brettchalupa** [comment]
Thanks a bunch! :tada:

**Pull request merged**